tests/python/relay/test_tensorrt.py

```diff
@@ -513,49 +536,53 @@ def check_trt_used(graph):
         'vgg16',
         'densenet169',
         'densenet201']
+
     if test_all_models:
         models.extend(additional_models)
-    
+
     dtype = 'float32'
     input_shape = (1, 3, 224, 224)
     i_data = np.random.uniform(-1, 1, input_shape).astype(dtype)
     for model in models:
         latency[model], res = test_model(model, i_data, input_shape, dtype, use_trt=True)
         _, ref_res = test_model(model, i_data, input_shape, dtype, use_trt=False, num_iteration=1)
         tvm.testing.assert_allclose(res.asnumpy(), ref_res.asnumpy(), rtol=1e-3, atol=1e-3)
-    
+
     for model in models:
         print(model, latency[model])
 
 def test_tensorrt_serialize():
```

Comment:
Let's make sure not to remove any previous tests, we should only be adding additional tests for VM. We still need the test for graph runtime serialization

Reply:
Thanks, I am adding the graph runtime codes as well